tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -340,7 +340,9 @@ const userSharesFileOrFolderWithUserOrGroupWithExpirationDate = async function (
     .closeSidebar(100)
     .openSharingDialog(resource)
 
-  return api.sharingDialog().shareWithUserOrGroup(sharee, shareWithGroup, 'Viewer', undefined, remote, days)
+  return api.sharingDialog().shareWithUserOrGroup(
```

Comment:
Suggestion: No need to refactor here as line 343 is visible without needing to scroll sideways. It just becomes confusing to review.

Reply:
done!